src/hooks/core/chat/useSystemMessage.ts

```diff
@@ -16,7 +16,7 @@ const useSystemMessage = () => {
   }
 
   function __isMessageForOtherUser(message: GetstreamMessage): boolean {
-    return message?.other_system_user === 'true' && isMe(message?.other_system_user);
+    return isMe(message?.other_system_user);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic in the `__isMessageForOtherUser` function has been simplified. However, it seems like there might be a logical error. The new code checks if the message is from 'me' (the current user), which contradicts the function name `__isMessageForOtherUser`. Please ensure that this change doesn't introduce any logical errors.
```diff
- return isMe(message?.other_system_user);
+ return !isMe(message?.other_system_user);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->